crates/kftray-server/Dockerfile

```
@@ -1,18 +1,26 @@
 FROM rust:1.90.0-alpine3.20 AS builder
 
-RUN apk add --no-cache musl-dev
+RUN apk add --no-cache musl-dev openssl-dev openssl-libs-static pkgconfig
 
+WORKDIR /workspace
 
-RUN USER=root cargo new --bin kftray-server
-WORKDIR /kftray-server
+# Copy workspace Cargo.toml and .cargo directory if it exists
+COPY Cargo.toml ./
+COPY .cargo ./.cargo 2>/dev/null || true
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)